### PR TITLE
DOC Add SearchableDropdownField to 5.2.0 changelog

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/silverstripe-admin/issues/1618

### CODE REVIEW

**Review by GuySartorelli — COMMENTED** [review]
Note: The branch this PR is targetting now has a CI action. This PR will need to be force-pushed to run the CI, and the CI must be green before merging.

File: en/04_Changelogs/5.2.0.md
Comment (ChloeSartorelli) [inline en/04_Changelogs/5.2.0.md]:
```suggestion
  - [New searchable dropdown fields](#searchable-dropdown-field)
```
"Component" doesn't really make sense here, since that makes it sound like it's just a react thing. And mentioning the class isn't useful for people who have never seen the class before (which will be anyone reading the changelog :p)

File: en/04_Changelogs/5.2.0.md
Comment (ChloeSartorelli) [inline en/04_Changelogs/5.2.0.md]:
```suggestion
### New searchable dropdown fields {#searchable-dropdown-field}
```

File: en/04_Changelogs/5.2.0.md
Comment (ChloeSartorelli) [inline en/04_Changelogs/5.2.0.md]: These aren't just react components - they're form fields.
```suggestion
A pair of new dropdown form fields have been added which are particularly useful for dropdowns with a large number of options.

[`SearchableDropdownField`](api:SilverStripe\Forms\SearchableDropdownField) is similar to [`DropdownField`](api:SilverStripe\Forms\DropdownField) which allows selecting a single option, and [`SearchableMultiDropdownField`](api:SilverStripe\Forms\SearchableMultiDropdownField) is similar to [`ListboxField`](api:SilverStripe\Forms\ListboxField) which allows selecting multiple options.

Both of these fields include a `setIsLazyLoaded()` method which will load a limited number of options at a time using an AJAX request matching what a user has typed in. There are quite a few options to customise these, including optionally using `SearchContext` to power the lazy-loaded search functionality. We encourage you to look at the API documentation for these new classes to see what you can do with them.
```

File: en/04_Changelogs/5.2.0.md
Comment (ChloeSartorelli) [inline en/04_Changelogs/5.2.0.md]:
```suggestion
Note that these are both powered by react components, and are only intended to be used within the CMS. If you want to use them on the front-end of your project you will need to provide your own templates and javascript implementation for them.
```

**Review by GuySartorelli — CHANGES_REQUESTED** [review]
I think we should have an entry in https://docs.silverstripe.org/en/5/developer_guides/forms/field_types/ as well, since there are some features that might not be immediately clear or that people might not even think to look for (such as allowing it to use `SearchContext`)

If you disagree, I won't fight you on it - but in that case it should at least get a mention in https://docs.silverstripe.org/en/5/developer_guides/forms/field_types/common_subclasses/

File: en/04_Changelogs/5.2.0.md
Comment (emteknetnz) [reply en/04_Changelogs/5.2.0.md]: Updated

File: en/04_Changelogs/5.2.0.md
Comment (emteknetnz) [reply en/04_Changelogs/5.2.0.md]: Updated

File: en/04_Changelogs/5.2.0.md
Comment (emteknetnz) [reply en/04_Changelogs/5.2.0.md]: Updated

File: en/04_Changelogs/5.2.0.md
Comment (emteknetnz) [reply en/04_Changelogs/5.2.0.md]: Updated

**Comment by GuySartorelli** [comment]
Please either make the changes requested in https://github.com/silverstripe/developer-docs/pull/412#pullrequestreview-1776234444 or explain why you think those changes should not be made.

**Comment by emteknetnz** [comment]
Have added a couple of quick one-liners for the form field subclasses

**Review by GuySartorelli — APPROVED** [review]
LGTM - will merge once functionality has been merged.

**Review by GuySartorelli — CHANGES_REQUESTED** [review]
There's a markdown lint failure

**Review by GuySartorelli — APPROVED** [review]
Looks good

**Pull request merged**
